.circleci/config.yml

```
@@ -1022,7 +1022,7 @@ jobs:
     parallelism: 20
     environment:
       EVM: << pipeline.parameters.evm-version >>
-      SOLTEST_FLAGS: --enforce-via-yul
+      SOLTEST_FLAGS: --enforce-via-yul --no-smt
```

> Also we could think about restricting this further... the flag really only affects the semantics tests, doesn't it? But most of it is cheap anyways, so the SMT tests are the most wasteful ones I guess.

How about this? I wonder if this would actually work given the space after `-t`.
```suggestion
      SOLTEST_FLAGS: --enforce-via-yul -t syntaxTests
```

---

``-t semanticsTests`` you mean? But yeah, not sure if it'd work :-). I also haven't checked yet whether the flag is really *only* relevant for the extracted semantics tests.

---

> `-t semanticsTests` you mean?

Right, fixed.

Also, I checked now and looks like the `--enforce-via-yul` flag is handled specifically by `SemanticTest.h`. Also the only place where the `compileViaYul` directive ever appears are tests in `semanticTests/`. So it should be safe.